app/routes/users.js

```diff
@@ -32,7 +32,7 @@ export default Route.extend(DataErrorMixin, CanonicalRedirectMixin, CoverPageMix
   },
 
   serialize(model) {
-    return { name: get(model, 'name') };
+    return { slug: get(model, 'slug') };
```

Comment:
Don't we also need to have `|| get(model, 'id')` to have it fallback to numerical ID in link generation?

Reply:
Yeah we should, but I don't think there is anywhere where we create a link without knowing the slug.

Reply:
slug can be missing if a user hasn't chosen one yet, so we probably need to handle that

Reply:
Really? Figured it was auto-set to their name since mine was on production API.

Reply:
That was just until we release the ability to pick slugs, it's not a permanent thing.